qa-common/README

```diff
@@ -0,0 +1,13 @@
+This directory has been created to contain common CI related functions
+shared across multiple CI jobs in multiple repos.
+
+When new CI job is being created it can be placed in:
+* repo .gitlab-ci file directly - this is reserved for repo specific jobs
+* in the mendertesting/templates - for complete CI components used across
```

Comment:
"CI/CD components" or "GitLab CI/CD components" - let's use the same name used in the official docs.

Reply:
Adjusted and updated.